codegen-plugin/src/main/kotlin/io/spine/protodata/hello/BuilderBeforeReturnInsertionPoint.kt

```diff
@@ -0,0 +1,87 @@
+package io.spine.protodata.hello
+
+import io.spine.protodata.renderer.InsertionPoint
+import io.spine.text.Text
+import io.spine.text.TextCoordinates
+import io.spine.text.TextFactory
+import org.jboss.forge.roaster.Roaster
+import org.jboss.forge.roaster.model.source.JavaClassSource
+import org.jboss.forge.roaster.model.source.JavaSource
+import org.jboss.forge.roaster.model.source.MethodSource
+import java.util.regex.Pattern
+
+/**
+ * [InsertionPoint] that is the line just before `return` statement
+ * in the `build` method of the message class builder.
+ */
+public class BuilderBeforeReturnInsertionPoint : InsertionPoint {
+
+    public override val label: String =
+        BuilderBeforeReturnInsertionPoint::class.java.simpleName
+
+    private val returnLinePattern = Pattern.compile(
+        "\\s*return .+;.*", Pattern.UNICODE_CASE or Pattern.DOTALL
+    )
+
+    /**
+     * Returns the position of the line just before the `return` statement
+     * of the `build` method of the message builder class. The builder class
```

Comment:
Usually, we have just a single sentence in the first paragraph. All the rest goes to subsequent paragraphs.

How about this:

```kotlin
/**
 * Returns the position of the line just before the `return` statement
 * of the `build` method of the message builder class. 
 *
 * The position is being searched within the given [text],
 * taking into account the following:
 * 
 *  - The builder class is the nested within class of some `Message`. 
 *  - The class of `Message` class is the only top-level class in the passed [text].
 * 
 * This implementation assumes that Proto file has its `java_multiple_files` option
 * set to `true`.
 */

Reply:
Nice one. I like it.